source/_docs/energy/gas.markdown

```diff
@@ -23,3 +23,9 @@ We have worked with creator [Marcel Zuidwijk](https://www.zuidwijk.com) to devel
 
 ![Photo of SlimmeLezer attached to a smart electricity meter](/images/docs/energy/slimmelezer.jpg)
 
+
+## Project: AI-on-the-Edge
```

Comment:
All titles above are descriptive of what it does (e.g., connect to your meter, connect using a P1 port), but this title seems out of bounds with that concept. Can we make it descriptive like the others?